src/pfe/portal/routes/projects/remoteBind.route.js

```diff
@@ -116,7 +116,11 @@ async function bindStart(req, res) {
     if (err.code === 'INVALID_PROJECT_NAME'){
       res.status(400).send(err.info);
     } else {
-      res.sendStatus(500);
+      let errorMessage = ""
+      if (err) {
+        errorMessage = err.info || err
+      }
+      res.status(500).send(`Project bind failed ${errorMessage}`);
```

Comment:
nit: I think this is equivalent to:
```suggestion
      res.status(500).send(`Project bind failed ${err.info || err}`);
```